packages/kbn-legacy-logging/src/log_format.ts

```diff
@@ -74,8 +82,8 @@ export default class TransformObjStream extends Stream.Transform {
     return date.format(format);
   }
 
-  readEvent(event) {
-    const data = {
+  readEvent(event: any) {
```

Comment:
Thank you for converting this file to TS that will simplify our migration to the KP logging system in the future. However, I'd suggest making an extra step and declare the Event interface.
```typescript
interface GenericEvent {
  event: string;
  timestamp: string;
  pid: string;
  tags?: string[];
}

interface OpsEvent {
  event: string;
  timestamp: string;
  pid: string;
  tags?: string[];
}

interface ResponseEvent {
  method: 'GET' | 'POST' | 'PUT' | 'DELETE';
  statusCode: number;
  path: string;
  headers: Record<string, string | string[]>;
  responsePayload: string;
  responseTime: string;
  query: Record<string, any>;
}

interface OpsEvent {
  pid: string;
  os: {
    load: string[];
  };
  proc: Record<string, any>;
  load: string;
}

interface ErrorEvent {
  error: Error;
  url: string;
}

interface LogEvent {
  data: EventData;
}

interface AnotherEvent {
  data: string | Record<string, any>;
}

// FIXME should be a union of types. we are going to improve type safety later
type Event = GenericEvent & ResponseEvent & OpsEvent & ErrorEvent & LogEvent & AnotherEvent;
```

Reply:
Done in https://github.com/elastic/kibana/pull/77678/commits/f2766e5c0a6b83ec8f189cf3c8d9b89d8b7c50a2. I could even introduce a union based type.